File: 0075-sort-colors/0075-sort-colors.java
class Solution {
    public void sortColors(int[] nums) {
        int z=0,o=0,t=0;
        for(int i :nums){
            if(i==0)
                z++;
            else if (i==1)
                o++;
            else
                t++;
            
        }for(int i=0;i<nums.length;i++){
            if (z>0){
                nums[i]=0;
                z--;
            }
                
            else if(o>0){
                nums[i]=1;
                o--;
            }
                
            else{
                nums[i]=2;
                t--;
            }
        }
    }
}

File: 0075-sort-colors/0075-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        low=0
        mid=0
        high=len(nums)-1
        while mid<=high:
            print(nums[mid])
            if nums[mid]==0:
                temp=nums[mid]
                nums[mid]=nums[low]
                nums[low]=temp
               
                mid+=1
                low+=1
            elif nums[mid]==1:
                mid+=1
            elif nums[mid]==2:
                temp=nums[mid]
                nums[mid]=nums[high]
                nums[high]=temp
                high-=1
        